app/scripts/controllers/containerdetails.js

```diff
@@ -198,7 +198,7 @@ angular.module('dockstore.ui')
           "\ndockstore tool convert entry2json --entry " + tool_path + ":" + $scope.toolTagName +" > Dockstore.json" +
           "\nvim Dockstore.json"+
           "\n# run it locally with the Dockstore CLI" +
-          "\ndockstore tool launch --entry " + tool_path + ":" + $scope.toolTagName + " --json Dockstore.json";
+          "\ndockstore tool launch --entry " + tool_path + ":" + $scope.toolTagName + " --json Dockstore.json --descriptor " + $scope.desc;
```

Comment:
If indeed, we default to CWL. We probably only need to add `--descriptor` when it is WDL, but not when it is CWL even if there are multiple descriptors available.